frontend/src/scenes/surveys/surveyLogic.tsx

```diff
@@ -717,7 +723,7 @@ export const surveyLogic = kea<surveyLogicType>([
                 // Initialize date range based on survey dates when survey is loaded
                 if ('created_at' in values.survey) {
                     const dateRange = {
-                        date_from: dayjs(values.survey.created_at).startOf('day').format(DATE_FORMAT),
+                        date_from: getSurveyStartDateForQuery(values.survey),
```

Comment:
so here it was using `created_at` only, but the new function uses start and created, with a preference for start, I believe this would be a bug?

Reply:
the code even check for the existence of `'created_at'`

Reply:
all the other dates have always used the `start_date` as the default date, this was the only one that was using only `created_at`. so it's more about doing the same behavior to this date range as well

Reply:
then i think you should remove the `if ('created_at' in values.survey) {` condition, which wouldn't make sense anymore?